### PR TITLE
Make all the CI `.sh` scripts have a consistent set of permissions

### PR DESCRIPTION
**Pull request by brycelelbach**
Some of the `.sh` scripts in `ci/` have executable file permissions. Some do not. They should all have a consistent set of file permissions. This change makes them all executable.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by harrism** [comment]
ok to test

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Review by raydouglass — APPROVED** [review]

**Comment by brycelelbach** [comment]
@harrism changelog added.

File: CHANGELOG.md [hunk 1]
```diff
@@ -192,6 +193,7 @@
 - PR #6445 Add `dlpack` to run section of libcudf conda recipe to fix downstream build issues
 - PR #6450 Make java Column Builder row agnostic
```
Comment (harrism) [inline CHANGELOG.md:195]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -168,6 +168,7 @@
 - PR #6281 Fix unreachable code warning in datetime.cuh
 - PR #6286 Fix `read_csv` `int32` overflow 
 - PR #6310 Replace a misspelled reference to `master` branch with `main` branch in a comment in changelog.sh
+- PR #6286 Fix `read_csv` `int32` overflow
```
Comment (kkraus14) [inline CHANGELOG.md:171]: Duplicate changelog entry
```suggestion
```

**Review by kkraus14 — APPROVED** [review]

**Pull request merged**